V1.0

FokkeDijkstra opened the pull request:
First release of jobinfo. With a fix for handling missing data for memory information and improved README file.

bedroge commented on README.md:
```diff
@@ -3,3 +3,26 @@ Tool for giving a quick overview of Slurm job statistics.
+  jobid        the jobid to query
+
+optional arguments:
+  -h, --help   show this help message and exit
+  -d, --debug  Shows extra information about how the data is gathered
+  -l, --long   Shows more information about the job
```
Nitpicking, but let's make this consistent:
```suggestion
  -h, --help   show this help message and exit
  -d, --debug  show extra information about how the data is gathered
  -l, --long   show more information about the job
```

FokkeDijkstra replied:
Fixed.

bedroge commented on README.md:
```diff
@@ -3,3 +3,26 @@ Tool for giving a quick overview of Slurm job statistics.
+
+# Release notes:
+
+## v1.0 - 2024-04-15
```
Shouldn't this be May?

FokkeDijkstra replied:
Thanks for spotting this. Fixed.

bedroge commented:
Don't know if I was supposed to review all code or just the changes from this PR, but I mostly did the latter, and also tried the new jobinfo on some jobs myself. This worked fine, only ran into one thing that looked a bit odd for job 8916761:
```
State                          : CANCELLED,COMPLETED,TIMEOUT  
```

By using sacct I found that this is actually a combination of all statuses (job, batch, step), and now it does make a bit more sense to me. I don't really know what it should report in such cases anyway, as it may always be a bit confusing. The old jobinfo reports:
```
State               : CANCELLED,TIMEOUT
```
:man_shrugging:

FokkeDijkstra commented:
Thanks for spotting the issue with the State information. I've now adjusted the code to reduce the items displayed for the State. I also replaced the uid with user or operator in the 'CANCELLED by' string.
